### ISSUE TITLE
EC 11.3.0 version-manifest.txt doesn't show 11.3.0

### ISSUE THREAD

**Issue by jeremiahsnapp** [chef/opscode-omnibus#718]
Instead its first line is `private-chef 11.2.6+20150209142733.git.6.46e29fc`.

This makes it hard to know what's installed when troubleshooting a customer problem.

**Comment by mmzyk** [comment]
This is on me, as I must have missed a step in the release process.

**Comment by jeremiahsnapp** [comment]
Cool, just making sure that's all it was. Thanks @mmzyk close this if you want.

**Comment by mmzyk** [comment]
@jeremiahsnapp I'm trying to figure out if this was a mistake on my part or if something didn't work properly.

For future reference however, that line still tells you a lot. 

private-chef 11.2.6+20150209142733.git.6.46e29fc
That is basically saying: this build is 6 comiits past 11.2.6 with a revision of 46e29fc

You can use that revision number to figure out the commit on github. In this case it resolves to one of the commits that was included in 11.3.0. The manifest should still say 11.3.0 however, so something went wrong.

**Comment by jeremiahsnapp** [comment]
Thanks, I knew about the revision part but I didn't know about the "6 commits past 11.2.6" part.

It could be because of something I'm not understanding regarding the git history but neither of those bit of information seem to point me to the actual tag for 11.3.0.

https://github.com/chef/opscode-omnibus/commits/11.3.0

But at least they are better indicators that it's not really 11.2.6 so it's handy for that.

**Comment by mmzyk** [comment]
@jeremiahsnapp Yeah, @stevendanna and I have been investigating to figure out what happened here, because I appear to have followed the correct process. There's something odd going on as that referenced sha in the manifest only shows up on github (perhaps one of their special things to keep track of PRs, etc). All the changes do show up locally, but under different shas.

**Comment by stevendanna** [comment]
Some additional info.  While 46e29fc exists on GitHub, we can't seem to find it on a local cloned repository:

```
> git show 46e29fc453d570aaa64b47cf41c63d9313d35a9f
fatal: bad object 46e29fc453d570aaa64b47cf41c63d9313d35a9f
```

**Comment by mmzyk** [comment]
All right, I'm going to close this out. I've looked at the code and it the way the manifest is generated seems okay and I can't find a bug there, although it should have created the manifest using the 11.3.0 tag. I've tried to determine where that orphaned commit came from on github and can't find it. I even brought back the branch that contained the changes from the dead and still couldn't find it.  I also did an adhoc build on the CI server and it created the manifest correctly.

The only other thing I can think of is that something special is going on with the release job, but that should not be the case. Guess we'll find out the next time a release is cut, but I think this was a solar flare caused blip.
